api/random_recipe.go

```diff
@@ -35,7 +33,7 @@ type Ingredient struct {
 // type Step struct {
 // 	Number int    `json:"number"`
 // 	Step   string `json:"step"`
-// }
+// }logToFile
```

Comment:
can we stop logging to a file entirely its prolly not worth it

Reply:
yeah that is why it is commented out I know you and Mary talked about it. :). Figured it could not hurt to show that we tried different things even if they were not all in the final project.